lib/studies/reply/adaptive_nav.dart

```diff
@@ -1120,40 +1121,22 @@ class _ReplyFabState extends State<_ReplyFab>
   }
 }
 
-class _FabSwitcher extends StatelessWidget {
-  const _FabSwitcher({@required this.onMailView}) : assert(onMailView != null);
-
-  final bool onMailView;
-
-  static final fabKey = UniqueKey();
-
-  @override
-  Widget build(BuildContext context) {
-    return AnimatedSwitcher(
-      duration: _kAnimationDuration,
-      transitionBuilder: (child, animation) => ScaleTransition(
-        child: child,
-        scale: animation,
-      ),
-      child: onMailView
-          ? Icon(Icons.reply_all, key: fabKey)
-          : const Icon(Icons.create),
-    );
-  }
-}
-
-class _InboxTransitionSwitcher extends StatelessWidget {
-  const _InboxTransitionSwitcher({@required this.child})
-      : assert(child != null);
+class _FadeThroughTransitionSwitcher extends StatelessWidget {
+  const _FadeThroughTransitionSwitcher({
+    @required this.child,
```

Comment:
Make child the last param to follow conventions

Reply:
Will do!